features/guidance/anticipate-lanes.feature

```diff
@@ -343,6 +343,38 @@ Feature: Turn Lane Guidance
             | waypoints | route    | turns                           | lanes                                  |
             | a,h       | ab,gh,gh | depart,roundabout-exit-5,arrive | ,slight right:false slight right:true, |
 
+    @anticipate @todo
+    Scenario: Anticipate with lanes in roundabout where we stay on the roundabout for multiple exits
+        # Scenario requires correct lane handling for clockwise roundabouts
+        Given the profile "lhs"
+        And the node map
+            |   |   | a |   |   |
+            |   |   | b |   |   |
+            | h | c |   | g |   |
+            |   |   |   |   |   |
+            |   | d |   | f |   |
+            |   |   | e |   |   |
+            | x |   |   |   | y |
+
+        And the ways
+            | nodes | turn:lanes:forward         | highway | junction   |
+            | ab    | slight_left\|slight_left   | primary |            |
+            | bg    |                            | primary | roundabout |
+            | gf    |                            | primary | roundabout |
+            | fe    |                            | primary | roundabout |
+            | ed    |                            | primary | roundabout |
+            | dc    | slight_left                | primary | roundabout |
+            | cb    |                            | primary | roundabout |
+            | ch    |                            | primary |            |
+            | ex    |                            | primary |            |
+            | dx    |                            | primary |            |
+            | gy    |                            | primary |            |
+            | fy    |                            | primary |            |
+
+        When I route I should get
+            | waypoints | route      | turns                           | lanes                                  |
+            | a,h       | ab,ch,ch   | depart,roundabout-exit-5,arrive | ,slight right:false slight right:true, |
```

Comment:
I think you forgot to change the expected `lanes` output here from `slight right` to slight left.
If you take a look at the way `ab` and `dc` above, they only have `slight left` lanes.

Counterclockwise vs clockwise does not change the turn lane's direction.
It only changes lane anticipation's behavior in keeping the user to the left/right.